src/test/java/com/exasol/adapter/dialects/AbstractSqlDialectTest.java

```diff
@@ -177,4 +183,50 @@ void testGetStringLiteralWithNull() {
         final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
         assertThat(sqlDialect.getStringLiteral(null), equalTo("NULL"));
     }
+
+    @Test
+    void testOmitParenthesesFalse() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        assertThat(sqlDialect.omitParentheses(ScalarFunction.ADD_DAYS), equalTo(false));
+    }
+
+    @Test
+    void testGetTableCatalogAndSchemaSeparator() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        assertThat(sqlDialect.getTableCatalogAndSchemaSeparator(), equalTo("."));
+    }
+
+    @Test
+    void testGetSqlGenerationVisitor() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        SqlGenerationContext context = new SqlGenerationContext("catalogName", "schemaName", false);
+        assertThat(sqlDialect.getSqlGenerationVisitor(context), instanceOf(SqlGenerationVisitor.class));
+    }
+
+    @Test
+    void testGetScalarFunctionAliases() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        assertThat(sqlDialect.getScalarFunctionAliases(), equalTo(new EnumMap<>(ScalarFunction.class)));
+    }
+
+    @Test
+    void testGetBinaryInfixFunctionAliases() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        assertAll(() -> Assert
+                    .assertThat(sqlDialect.getBinaryInfixFunctionAliases().get(ScalarFunction.ADD), Matchers.equalTo(
+                          "+")), //
+              () -> Assert.assertThat(sqlDialect.getBinaryInfixFunctionAliases().get(ScalarFunction.SUB),
+                    Matchers.equalTo("-")), //
+              () -> Assert.assertThat(sqlDialect.getBinaryInfixFunctionAliases().get(ScalarFunction.MULT),
+                    Matchers.equalTo("*")), //
+              () -> Assert.assertThat(sqlDialect.getBinaryInfixFunctionAliases().get(ScalarFunction.FLOAT_DIV),
+                    Matchers.equalTo("/")));
+    }
+
+    @Test
+    void testGetPrefixFunctionAliases() {
+        final SqlDialect sqlDialect = new DummySqlDialect(null, AdapterProperties.emptyProperties());
+        assertAll(() -> Assert
```

Comment:
There is only one prefix. Please remove `assertAll` wrapper.

Reply:
done